Updated Hal Finkel DOE video link in funder series

bjoyce3 opened the pull request:
<!--- Thank you for opening a pull request! Here are some helpful tips:
     
      1. To solicit reviewers: 
           the @usrse-maintainers are automatically notified when you open this pull request
           If you need additional reviewers you can:
               (if you have permission to do so) assign the label "reviewers-needed" 
               if you are on the usrse slack, post a link to your PR there and ask for reviewers

      2. To get help:
           you can ask the question directly in this pull request for @usrse-maintainers
           you can ask a question in the #website channel of usrse.slack.com
           for important issues, you can @usrse-admin to alert all admins of the repository (use sparingly)
 -->

<!--- Provide a general summary of your changes in the Title above -->

## Description
- Changed the WG Outreach Funder Series pages to include Hal Finkel video recording
- Removed registration information from the event page now that the event has passed

## Motivation and Context
Finalizing the event page and communicating the talk to wider RSE community.

## Checklist:
<!--- Go over all the following points, and put an `x` in all the boxes that apply. -->
<!--- If you're unsure about any of these, don't hesitate to ask. We're here to help! -->
- [ ] I have posted the link for the PR in the usrse slack (#website) to ask for reviewers
--- I haven't done this yet because I'm still creating the PR...but I will.
- [ x] I have previewed changes locally
- [ ] I have updated the README.md if necessary

cc @usrse-maintainers

danielskatz commented:
can we share the slides as well as the video on both pages?

bjoyce3 commented:
I thought Hal gave permission for the video; didn't realize he gave permission for the slides as well.

danielskatz commented:
Yes, he sent the slides and gave permission. I'll forward them to you in case I haven't before

bjoyce3 commented:
@danielskatz let's put the slides in the Google Drive? We've got a folder for the recordings. I just created one for slides.

danielskatz commented:
sounds good - can you do this with the slides I emailed?

bjoyce3 commented:
Done and done. Any idea where we normally put slides in the repo? I see an /assets/images folder. They're all .pngs but a .pdf image not soooo different maybe.

cosden commented:
@bjoyce3 - I don't think we usually host the pdfs, but would instead link to them elsewhere (like a folder in the US-RSE google drive perhaps - I could create with view only permissions). I'm not opposed to hosting them in the repo, but I just wouldn't want the repo to get too big with loads of pdfs.

bjoyce3 commented:
Ok! Added links to slides on the two pages and modified the contributor guide to outline where some of the website stuff actually is and that we have a place to put speaker slides for the time being (and how to find it).

mrmundt commented on _events/2022/2022-10-funder-talk-series.md:
```diff
@@ -18,18 +18,14 @@ US-RSE is happy to be starting a new series of conversations with funders for 1)
-Please join us in this interactive conversation. (If you can't participate in the interactive event, where you will be able to ask questions and help funders understand US-RSE, it will also be recorded.)
-
-[**Register on Zoom**](https://boisestate.zoom.us/meeting/register/tJckd-2vqjIoGNRrjGPruBRxQBthv5Maidmm)
+The inaugural talk of the US-RSE Funder Talk Series will be with Hal Finkel, Ph.D. from the US Department of Energy's (DOE's) Office of Advanced Scientific Computing Research (ASCR). This event took place **Monday, October 17th at 2 PM ET, 1 PM CT, 12 PM MT, 11 AM PT**
```
```suggestion
The inaugural talk of the US-RSE Funder Talk Series featured Hal Finkel, Ph.D. from the US Department of Energy's (DOE's) Office of Advanced Scientific Computing Research (ASCR). This event took place **Monday, October 17th at 2 PM ET, 1 PM CT, 12 PM MT, 11 AM PT**
```

mrmundt replied:
Just one language thing - otherwise, fabulous!

bjoyce3 replied:
Committed. No idea why this is still listed as 'change suggested' below. I checked before the last request and it was changed to featured.

danielskatz commented:
previews: https://output.circle-artifacts.com/output/job/46e874a2-b536-4bfb-8abc-3a323622b918/artifacts/0/usrse.github.io/events/2022/2022-10-funder-talk-series/index.html and https://output.circle-artifacts.com/output/job/46e874a2-b536-4bfb-8abc-3a323622b918/artifacts/0/usrse.github.io/wg/outreach/index.html

danielskatz commented on pages/wg/outreach.md:
```diff
@@ -75,6 +75,10 @@ a speaker series aimed at bringing representatives from funders (e.g., NSF, NIH)
 to speak to members of US-RSE. These funders will bring essential information
 about opportunities, strategies, and starting points for finding funding.
 
+- __Hal Finkel__ - Advanced Scientific Computing Research, Office of Science, US Department of Energy
```
```suggestion
- __Hal Finkel__ - Advanced Scientific Computing Research, Office of Science, US Department of Energy: [video](https://youtu.be/vZYQJ5eOMls) & [slides](https://drive.google.com/file/d/1W9Oh50BMwHG6o1OEmlRXi_AaJYmMdMVn/view?usp=sharing)
```

danielskatz replied:
just trying to make this a bit more compact, with the idea that we will one day have lots of rows here.  If you accept this, you would also want to delete the next two lines

bjoyce3 commented:
Trying to figure out how to look at the clean copy now that I've added both suggestion...

bjoyce3 commented:
Aight @danielskatz or @mrmundt I believe both suggestions have been incorporated and the lines deleted as needed.

danielskatz commented:
I'm not sure why it's still marked that way either - I'll take a last look at the preview pages now, and then merge anyhow if they look ok

danielskatz commented:
For the info on the outreach WG page on the talk by Hal, let's add the date too, as that will be helpful when we have more talks

danielskatz commented:
other than adding the date, the preview pages look good to me

bjoyce3 commented:
WG page updated with the international standard date for the talk.

danielskatz reviewed (approved):
great!
